builder/amazon/chroot/block_device.go

```diff
@@ -1,69 +0,0 @@
-//go:generate struct-markdown
-
-package chroot
-
-import (
-	"fmt"
-
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/service/ec2"
-	awscommon "github.com/hashicorp/packer/builder/amazon/common"
-	"github.com/hashicorp/packer/template/interpolate"
-)
-
-type BlockDevice struct {
-	awscommon.BlockDevice `mapstructure:",squash"`
-	// ID, alias or ARN of the KMS key to use for boot volume encryption. This
-	// only applies to the main region, other regions where the AMI will be
-	// copied will be encrypted by the default EBS KMS key. For valid formats
-	// see KmsKeyId in the [AWS API docs -
-	// CopyImage](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CopyImage.html)
-	// This field is validated by Packer, when using an alias, you will have to
-	// prefix kms_key_id with alias/.
-	KmsKeyId string `mapstructure:"kms_key_id" required:"false"`
```

Comment:
awscommon.BlockDevice already had a `KmsKeyId` field ( this was causing some issues with code generation); so I removed this definition here.